test/output_formatter_test.cc

```diff
@@ -224,5 +225,12 @@ TEST_F(StatidToNameTest, TestTranslations) {
   }
 }
 
+TEST_F(MediumOutputCollectorTest, FortioPedanticFormatter) {
+  const auto input_proto = loadProtoFromFile("test/test_data/output_formatter.medium.proto.gold");
```

Comment:
The "auto" here doesn't seem to hide redundant type information, so it might be better for the reader to spell it out.

Reply:
Done here and in other spots in 2df8c2be